libbeat/tests/system/beat/beat.py

```diff
@@ -207,8 +207,11 @@ def render_config_template(self, template_name=None,
 
         kargs["beat"] = self
         output_str = template.render(**kargs)
+
+        prev_umask = os.umask(0177)
```

Comment:
@andrewkroh Could you have a look at this change? WDYT?

Reply:
Rather than setting a temporary umask, how about we write the file with the permissions that we want. Can we do something like this: https://stackoverflow.com/a/5624691/503798 ?

Reply:
Yes, better doing it explicitely, doing it with a chmod, I think it's more clear this way.